build.psm1

```diff
@@ -1374,7 +1374,7 @@ function Test-PSPesterResults
 
 function Start-PSxUnit {
     [CmdletBinding()]param(
-        [string] $ParallelTestResultsFile = "ParallelXUnitResults.xml"
+        [string] $xUnitTestResultsFile = "xUnitResults.xml"
```

Comment:
Please update the YML to reflect the new name

Reply:
https://github.com/PowerShell/PowerShell/blob/57aeca428eba5f2ec3579562d74e9aac4bbaf615/.vsts-ci/windows.yml#L93